content/en/blog/general/introducing-kyverno-envoy-plugin/index.md

```diff
@@ -0,0 +1,236 @@
+---
+date: 2024-06-04
+title: Kyverno-Envoy-Plugin - Kyverno policies based authorization plugin for Envoy 
+linkTitle: Kyverno-Envoy-Plugin - Kyverno policies based authorization plugin for Envoy
+author: Sanskar Gurdasani
+description: Make external authorization easy with Kyverno-Envoy-Plugin.
+---
+
+![Kyverno-Envoy-Plugin](logo.png)
+
+While microservices offer benefits by breaking applications into smaller, independent parts, they also introduce new challenges offcourse microservies improve productivity of individual development. Managing service discovery, authentication, and authorization across multiple microservices can be complex. These tasks become even more intricate due to the diverse and short-lived nature of microservice environments.
```

Comment:
offcourse?

Reply:
Shall we add a rephrased version of the above lines?
```
Microservices enhance the productivity of individual development teams by dividing applications into smaller, independent components. However, microservices alone do not address longstanding challenges in distributed systems such as authentication and authorization. These problems can become even harder to manage due to diverse and short-lived nature of the microservice environments.
```
Or can you suggest something to start the blog?